### ISSUE TITLE
Unable to add group admin people

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#638]
new site
logged in as admin

@pmackay I am trying to add someone in to the admin status for Brockwell Lido Users group. Shelley silas needs to be made an admin person. I can see there is a request pending to be a member:

![image](https://cloud.githubusercontent.com/assets/6060985/7101596/439cd6b6-e058-11e4-8e01-71a563ddafab.png)

I have tried to edit her status to be an admin person but cannot figure out why it doesn't update. In the group when you look at members she is listed there. Now stuck.

**Comment by pmackay** [comment]
What exact steps did you try? 

I just made her active by:
- ticking left col tickbox
- selected "Modify membership status" under Choose an operation.
- Execute
- Change to Active in dropdown
- Next
- Confirm

Status changed...

**Comment by GilesGibson** [comment]
Hmm, I tried round that area of menus. I went through and did the select user and then had the drop down options of "Active, pending, blocked". Is this the way to make someone a member? So "active" means being a member? So what does pending mean? Will that be if they have asked just to be a member of the group? So making them active means they are a member. If so I will a) update the admin manual and b) shoot the drupal coder who uses this sort of language.
